### ISSUE TITLE
Tagging multiple threads unable to apply tags

### ISSUE THREAD

**Issue by manishrjain** [astroidmail/astroid#399]
https://github.com/astroidmail/astroid/wiki/Tagging-multiple-threads

Selected multiple threads, pressed `+` to bring up the tags, modified them, pressed enter. Nothing happens. Then pressed `l`, just moves to the next tab. So, unable to modify tags of multiple tagged threads. Keybindings doesn't seem to use `l` key for anything; or has any setting for applying tags to tagged threads.

**Comment by gauteh** [comment]
Check ? for correct keybindings. Search for previous issue. Wiki needs
update.

lør. 9. sep. 2017 kl. 05.28 skrev Manish R Jain <notifications@github.com>:

**Comment by manishrjain** [comment]
It's not clear from the keybindings, which is what I was looking at. Also, the keybinding tab, doesn't allow any search on it, which makes it super hard to follow.

But, I was able to resolve this one by going through the past issues. I've updated the wiki page. It required a `;`, followed by an action.
